docs/flow/tutorials/collecting-prometheus-metrics.md

```diff
@@ -0,0 +1,65 @@
+---
+aliases:
+- /docs/agent/latest/flow/tutorials/collecting-prometheus-metrics
+title: Collecting Prometheus metrics
+weight: 200
+---
+
+Grafana Agent is a telemetry collector and the primary goal is to get telemetry data from one location to another. The goal of this tutorial is to get a basic working example of Grafana Agent working in Flow mode.  
+
+# Prerequisites
+
+* [Docker](https://www.docker.com/products/docker-desktop)
+* Clone the [Agent Repository](https://github.com/grafana/agent) `git clone git@github.com:grafana/agent.git`
```

Comment:
Should the 'git clone...' command be on its own line?

Reply:
Makes it easier to copy and paste so fine with changing it.